### PR TITLE
Feat/add colors to team members zappy gui

### PR DESCRIPTION
**Pull request by Thomaltarix**
I added colors to the team and team members.

> [!Note]
> When you click on a player, the HUD takes the color of the team of the player

I also added the colors to the icons of the game HUD

### CODE REVIEW

File: gui/include/GameDatas/Team.hpp [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Gui::Team {
          * @param playerModelPath Path to the team model asset for players.
          * @param eggSkinPath Path to the skin of the team.
          */
-        Team(const std::string &name, const std::string &playerModelPath, const std::string &eggModelPath);
+        Team(const std::string &name, const std::string &playerModelPath, const std::string &eggModelPath, Color playerColor);
```
Comment (Njord201) [inline gui/include/GameDatas/Team.hpp:39]: Describe the added parameter in documentation.

**Review by Njord201 — CHANGES_REQUESTED** [review]

**Review by mathieurobert1 — APPROVED** [review]
Perfect just a request from @Njord201 about param update in function documentation.

**Review by Njord201 — DISMISSED** [review]

File: gui/include/GameDatas/Team.hpp (hunk 1 above)
Comment (Thomaltarix) [reply gui/include/GameDatas/Team.hpp:40]: Done

**Review by mathieurobert1 — APPROVED** [review]
:+1:

**Review by Njord201 — APPROVED** [review]

**Pull request merged**